### DISCUSSION TITLE
Linux free license?

### DISCUSSION THREAD

**Discussion by aracnotech** [xoureldeen/Vectras-VM-Android#93] [General]
Hi, I'm curious to ask you one thing, in rom section you have removed everything due to copyright issues, but Linux distros are not free distribution and copyright free?

**Comment by xoureldeen** [comment]
we dont have time to upload roms again, u must wait :)

**Reply by aracnotech** [reply]
Of course, I was just curious. Thanks for all your hard work

**Comment by xoureldeen** [comment]
u can also share ur cvbi files to us to upload it in our store and website with ur credits :)

**Reply by aracnotech** [reply]
Thanks
